run_tests.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 #Run all test classes
-phpunit --configuration phpunit.xml.dist
+vendor/bin/phpunit
```

Comment:
With this simple of a script, it seems it should just be deleted, and a line added to the README to that run tests just use phpunit. This also makes sure that if phpunit is installed globally or up the tree, you don't need the redundant copy.